src/main/java/alvioneurope/gsergeev/javagroup/clientlogimporter/ExecutedCommand.java

```diff
@@ -11,5 +15,80 @@
     private String operationId;
     private int responseLength;
     private int execTime;
+    private boolean executedCommand;
+
+    public ExecutedCommand(String executedCommand){
+        this.executedCommand = splitString(executedCommand);
+    }
+
+    public boolean checkString(String string) {
```

Comment:
1) I didn't understand what check this method 
2) If method return boolean value "is" should be used at the start